### ISSUE TITLE
Accessibility for Email Form Block

### ISSUE THREAD

**Issue by r-sauers** [r-sauers/contact-congress-plugin#60]
Accessibility for the Email Form Block is subpar. Here's what's missing:

1. The block does not form a distinct aria region, I'm unsure the proper way to do this yet. This is important because it should be known that the multiple forms involved are all related to each other.
2. It is good practice for the address bar to have an "aria-label" attribute.
3. The find representatives form fails to notify the screen reader about loading, success, and failure states. The aria-live, aria-busy, and aria-errormessage attributes should be used.
4. The Show Preview button should utilize the aria-pressed role.
5. There needs to be an association between Show Preview and the textarea, not sure what roles would work well for this. Probably aria-controls.
6. Representatives should perhaps be a distinct region.
7. Representative send email should have the aria-details attribute that point to the representative's information (aria-described-by might work too).
8. Maybe use aria-current, aria-controls, and aria-details to associate the send button beneath the form with the appropriate representative?
9. The textarea needs an aria-label
10. The whole form for the email template needs aria-label elements
11. It may be appropriate for the name fields to have aria-controls on the textarea?

**Comment by r-sauers** [comment]
Regarding 3, there is an <output> tag that would do wonders.
